setup-jest.ts

```diff
@@ -1,11 +1,14 @@
 import '@testing-library/jest-dom';
 import { toHaveNoViolations } from 'jest-axe';
 import 'jest-preset-angular/setup-jest';
+import { TextDecoder, TextEncoder } from 'util';
 
 expect.extend(toHaveNoViolations);
 
 Element.prototype.scrollIntoView = () => {};
 
+Object.assign(global, { TextDecoder, TextEncoder });
```

Comment:
why do we need this? Maybe add a comment here

Reply:
done